_data/sites.json

```
@@ -4669,6 +4689,16 @@
         ]
     },
 
+    {
+        "name": "GAINSFIRE",
+        "url": "https://www.gainsfire.com/",
+        "difficulty": "medium",
```

```suggestion
        "difficulty": "easy",
```
Given the description, this one sounds like it should be easy, no?